public/js/maps.js

```diff
@@ -42,12 +43,25 @@ function createMarkers(data) {
 
     google.maps.event.addListener(marker, 'click', function () {
       // Close current marker window if there is one open.
-      if (openMarkerWindow) openMarkerWindow.close();
+      if (openMarkerWindow) { 
+        openMarkerWindow.close(); 
+      }
```

Comment:
Do these lines need to be below line 49?

Reply:
When you click on a marker, you want to close the last marker before opening the new one (in order to only open 1 marker at a time). What's your logic for moving it below line 49?

Reply:
Makes sense! Wasn't looking at the context before, I just saw that `openMarkerWindow` was assigned after its value was checked and was wondering if that was the desired functionality.